prisma/migrations/20230114114951_specialties/migration.sql

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Reminder that migrations are generated files and will not be hand-edited. Please see `schema.prisma` for a clearer view of the changes